tests/e2e/README.md

```diff
@@ -17,20 +17,33 @@ Run `make test-e2e` to execute the tests.
 | `project create` | Y |
 | `project get` | Y |
 | `project delete` | Y |
+| `project list` | N |
+| `project set` | N |
 | `app create`  | Y  |
 | `app get`  | Y  |
 | `app set` | Y |
 | `app delete` | Y |
+| `app describe` | N |
 | `app list` | Y |
-| `catalog` | N |
-| `create --git` | N |
+| `catalog list` | Y |
+| `catalog search` | N |
+| `component get` | Y |
+| `component set` | Y |
+| `create --git` | Y |
```

Comment:
this is currently in progress in https://github.com/redhat-developer/odo/pull/437